Documentation/migration-scripts/python-scripts/repush_user_kafka_messages.py

```
@@ -0,0 +1,321 @@
+                    JOIN organizations o
+                        ON o.code = uo.organization_code AND o.tenant_code = uo.tenant_code
+                    WHERE uo.user_id = {user_id}
+                    ) org_with_roles
+                )
+                ) as merged
+                FROM user_organizations uo
+                LEFT JOIN organization_user_invites oui ON oui.username = '{username}'
+                LEFT JOIN invitations inv ON inv.id = oui.invitation_id
+                WHERE uo.user_id = {user_id}
+                LIMIT 1;
+            """
```

Ruff (0.15.1)

[error] Possible SQL injection vector through string-based query construction

(S608)